Tests/ParseTests.cs

```diff
@@ -143,6 +143,13 @@ public void TestSyntaxError7()
 			Assert.That(mod.ParseErrors.Count, Is.EqualTo(0));
 		}
 
+		[Test]
+		public void TestSyntaxError8()
+		{
+			var mod = DParser.ParseString(@"static if (is(__traits(parent, A) : __traits(parent, B))) {}");
```

Comment:
This is the last parse error I noticed while browsing all phobos files.

Reply:
Wow. Danke fürs fixen!!